container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -18,8 +18,6 @@ dnf_remove() {
   dnf remove -y \
       python3-devel \
       libcurl-devel \
-      gcc \
```

Comment:
Need to not remove more.

Reply:
Per discussion the relative gain is small for not keeping smaller stiff, so just removing the whole statement for now.